### ISSUE TITLE
Suggestion for graphics

### ISSUE THREAD

**Issue by holaimscott** [blashy101/Watermelon-Game-English-Patch#1]
Amazing work on the translation patches!

Been loving this game so much, and I am glad to see there is an english patch.

I just wanted to volunteer in case you needed to do any updates to the patches. I can do graphic design and try to make it match as close as how the game's original graphics look like.

I have experience editing textures for some other games and how sometimes the space is limited to sizes. Attached here are some mockups I could make if you are interested in any help.

Again, thank you for this patch, currently playing the game :)

<img width="1939" alt="watermelon01" src="https://github.com/blashy101/Watermelon-Game-English-Patch/assets/101499382/b0a4b797-6d73-4f20-bd84-8656c4c21d97">
<img width="1941" alt="watermelon02" src="https://github.com/blashy101/Watermelon-Game-English-Patch/assets/101499382/c0a860a6-ac58-4a81-84a2-d0e28b94b09b">
![icon](https://github.com/blashy101/Watermelon-Game-English-Patch/assets/101499382/ab7182a5-737c-484b-b362-d256eb6c9e58)

**Comment by blashy101** [comment]
Hi Scott! Thanks for the kind words. Your mock ups look phenomenal!! I’d
love to collaborate if you’re interested! I can send you art assets,
original Japanese ones and mine so you can see the differences if you’re
interested in overhauling!

I’ve got two other translations in the pipeline as well if you ever wish to
work together further. I can’t offer much but my undying gratitude, credit
online and my deepest appreciation.

All the best,

blashy

**Comment by holaimscott** [comment]
Absolutely, you can reach to me at my email at holaimscott@protonmail.com ;)

Apologies for the delay, I got busy IRL but mow I am back online
